components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { useState, useCallback, useRef, useEffect } from 'react';
+import {
+    Typography,
+    Toolbar,
+    ListItemText,
+    Menu,
+    ToolbarProps,
+    createStyles,
+    makeStyles,
+    useTheme,
+    Theme,
+    MenuProps as standardMenuProps,
+} from '@material-ui/core';
+import { ArrowDropDown } from '@material-ui/icons';
+import clsx from 'clsx';
+import { NavItem, DrawerNavGroup } from '../Drawer';
+import PropTypes from 'prop-types';
+
+export type ToolbarMenuItem = NavItem;
```

Comment:
Turns out we actually do need to do this to make the itemID prop optional. If you change this back, you can go and remove all the itemIDs that changing this forced you to add.

```suggestion
export type ToolbarMenuItem = Omit<NavItem, 'itemID'> & { itemID?: string };;
```